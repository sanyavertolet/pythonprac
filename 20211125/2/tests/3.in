ïóòü ìåæ êàìíåé ìàðøðóò íåëèíåéíûé ìû ïåðåæèâåì ýòî ñìóòíîå âðåìÿ
