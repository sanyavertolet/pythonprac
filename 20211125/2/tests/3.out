путь меж камней маршрут нелинейный мы переживем это смутное время
